src/v1/sources/shopify/webhookTransformations/serverSideTransform.js

```diff
@@ -111,11 +113,16 @@ const processEvent = async (inputEvent, metricMetadata) => {
   // attach anonymousId if the event is track event using note_attributes
   if (message.type !== EventType.IDENTIFY) {
```

Comment:
do you need any of post transformation steps for identify event ?
```suggestion
  if (message.type == EventType.IDENTIFY) return message;
```

Reply:
yes, handling common properties (like adding topic, shopify details and warehouse-related json paths) as well as after the redis setting for userId, we use that userId in all events whether identify or track

Reply:
And why not trigger `updateAnonymousIdToUserIdInRedis` in case of identify as we extended the signature of identify call to capture userId too now

Reply:
identify events do not have cartToken, so getting anonId is not possible